lib/ts/Diags.cc

```diff
@@ -716,14 +722,12 @@ Diags::should_roll_outputlog()
   bool ret_val = false;
   bool need_consider_stderr = true;
 
-  /*
   log_log_trace("should_roll_outputlog() was called\n");
-  log_log_trace("rolling_enabled = %d, output_rolling_size = %d, output_rolling_interval = %d\n", outputlog_rolling_enabled,
-                outputlog_rolling_size, outputlog_rolling_interval);
-  log_log_trace("RollingEnabledValues::ROLL_ON_TIME = %d\n", RollingEnabledValues::ROLL_ON_TIME);
-  log_log_trace("time(0) - last_roll_time = %d\n", time(0) - outputlog_time_last_roll);
-  log_log_trace("stdout_log = %p\n", stdout_log);
-  */
+  log_log_trace("should_roll_outputlog(): rolling_enabled = %d, output_rolling_size = %d, output_rolling_interval = %d\n",
```

Comment:
Yeah I was thinking about that when I first wrote the function. I'm not sure what the best way to do it would be. Right now it's compile time defined with a `#define`.